setup/updating.md

```diff
@@ -65,8 +65,47 @@ To update Coder, follow these steps:
    ```console
    helm upgrade --namespace coder --install --atomic --wait \
      --version 1.16.1 coder coder/coder --values current-values.yaml
+
    ```
 
+1. [Update your Helm chart](../guides/admin/helm-charts.md) via the steps below.
+Refer to the `1.21` [helm chart](https://github.com/cdr/enterprise-helm/blob/main/values.yaml)
+when making such changes:
+
+   Rename all instances of `cemanager` to `coderd`.
+
+   Remove all `dashboard.*` values. Route any traffic to `coderd`.
+
+   Remove all `envproxy.*` values. Route any traffic to `coderd`.
+
```

Comment:
SSH and cemanager.accessurl are also no longer managed by the helm chart.

SSH is done in the Manage > Providers page. The old value used to be [ssh.enable](https://github.com/cdr/enterprise-helm/commit/2cf1185503b7cf06f2097269a95eb460348aa657#diff-3c79bd4a49240f41e502e8e5ff347991776222adb25b3c5fe6386ac94c31e116L66)

Access URL is now just in Manage > Admin under the infrastructure page.